.github/workflows/osrm-backend.yml

```diff
@@ -536,21 +536,13 @@ jobs:
           echo "PKG_CONFIG_PATH=${OSRM_INSTALL_DIR}/lib/pkgconfig" >> $GITHUB_ENV
         fi
         popd
-    - name: Build example
```

Comment:
@DennisOSRM just out of curiosity: do we now consider libosrm as something completely internal? I.e. do not guarantee backward compatibility between releases etc? I had some ideas of various performance optimisations (what I am care a lot recently as you may noticed 😄) which could significantly complicate libosrm API, but if it would be completely internal it would ease things a lot 🤔

Reply:
It seems there are no production environments that use libosrm directly. it's one of those features that make sense on the surface of it, but in the end get zero usage. Happy to be convinced otherwise, tho.